Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,151 @@
+#Basic copy cat design of the mood file.
+#Figured I shouldn't fix something that isn't broken.
+#Shout out to whoever did it right the first time.
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_imhaving",category=['you'],prompt="I'm Having",pool=True,unlocked=True))
```

Comment:
To anyone who doesn't know better, "I'm having" might not immediately correlate to food, which could be confusing at first.